opus_codec_registration.src.html

```diff
@@ -132,13 +132,21 @@
 dictionary OpusEncoderConfig {
   OpusBitstreamFormat format = "opus";
   [EnforceRange] unsigned long frameDuration;
+  [EnforceRange] unsigned long complexity;
+  [EnforceRange] unsigned long packetlossperc;
+  boolean useinbandfec;
+  boolean usedtx;
 };
 </xmp>
 </pre>
 
 To check if an {{OpusEncoderConfig}} is valid, run these steps:
 1. If {{OpusEncoderConfig/frameDuration}} is not a valid `ptime` value,
     which is described in Section 6.1 of [[RFC7587]], return `false`.
+1. If {{OpusEncoderConfig/complexity}} is specified and not within the range of
```

Comment:
Since dictionary members are optional by default, is `If {{member}} is specified and ...` to be omitted?

Similar question for specifying the lower bound of `0`, when the `[EnforceRange] unsigned long` guarantees the member cannot be a negative number.

Reply:
precision never hurts, so I think it's fine to be a bit more verbose.

Reply:
Thanks, leaving it as is.